internal/controllers/machinedeployment/mdutil/util.go

```diff
@@ -564,7 +564,14 @@ func NewMSNewReplicas(deployment *clusterv1.MachineDeployment, allMSs []*cluster
 		scaleUpCount := *(deployment.Spec.Replicas) - currentMachineCount
 		return newMSReplicas + scaleUpCount, nil
 	case clusterv1.InPlaceMachineDeploymentStrategyType:
-		return 0, nil
+		// Find the total number of machines
```

Comment:
Isn't this the same as OnDelete above, meaning we could just have the InPlace case added to the above one in one line?

Reply:
yep, it is. I can move it up